_pkgdown.yml

```diff
@@ -15,6 +15,9 @@ repo:
 news:
   cran_dates: true
 
+development:
+  mode: auto
```

Comment:
@ddsjoberg Is this all thats necessary to fire up the development version of the website?

Reply:
Yep! But before we add this, let's have a conversation with @dgrassellyb ! We don't want to unnecessarily add more sites to our gh-pages. There is still some refactoring to do in the pkgdown workflow that builds the sites.

Reply:
lets remove